doc: Add changelog entry for BT APIs converting err to string (#16745)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

-------------

* The correct SoftDevice Controller library :kconfig:option:`CONFIG_BT_LL_SOFTDEVICE_MULTIROLE` will now be selected automatically when using coexistence based on :kconfig:option:`CONFIG_MPSL_CX` for nRF52-series devices.
* Added the APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` to allow printing error codes as strings.

>   The APIs return string representations of the error codes when :kconfig:option:`CONFIG_BT_HCI_ERR_TO_STR` and :kconfig:option:`CONFIG_BT_SECURITY_ERR_TO_STR` are enabled.

Reviewer: ```suggestion
  Each API returns string representations of the error codes when the corresponding Kconfig option, :kconfig:option:`CONFIG_BT_HCI_ERR_TO_STR` or :kconfig:option:`CONFIG_BT_SECURITY_ERR_TO_STR` Kconfig, is enabled.
```

>   The Bluetooth samples and :ref:`nrf53_audio_app` are updated to utilize these new APIs.

Reviewer: ```suggestion
  The :ref:`ble_samples` and :ref:`nrf53_audio_app` are updated to utilize these new APIs.
```

nRF5340 Audio
-------------

* Added:

>   * The APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` are used to allow printing error codes as strings.

Reviewer: ```suggestion
  * The APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` that are used to allow printing error codes as strings.
```
